src/sentry/static/sentry/app/components/events/interfaces/spans/utils.tsx

```diff
@@ -250,7 +250,10 @@ export const getHumanDuration = (duration: number): string => {
   // note: duration is assumed to be in seconds
 
   const durationMS = duration * 1000;
-  return `${Number(durationMS.toFixed(2)).toLocaleString()}ms`;
+  return `${Number(durationMS.toFixed(2)).toLocaleString(undefined, {
```

Comment:
Do you need to check that the options argument is supported? The docs indicate that options isn't widely available yet.

Reply:
Seems fine https://caniuse.com/#feat=mdn-javascript_builtins_date_tolocalestring_options

Reply:
Removed in https://github.com/getsentry/sentry/pull/20277/commits/50cab515f54097ff4637db45b4ecccbc5549a953